shared/chat/conversation/messages/wrapper/unfurl/prompt-list/container.tsx

```diff
@@ -11,74 +10,56 @@ type OwnProps = {
   ordinal: Types.Ordinal
 }
 
-const noPrompts = I.Set()
+const noPrompts = new Set<string>()
 const noMessageID = Types.numberToMessageID(0)
 
-const mapStateToProps = (state, {conversationIDKey, ordinal}: OwnProps) => {
-  const message = Constants.getMessage(state, conversationIDKey, ordinal)
-  const messageID = message && message.type === 'text' ? message.id : noMessageID
-  const promptDomains = state.chat2.unfurlPromptMap.getIn([conversationIDKey, messageID]) || noPrompts
-  return {
-    messageID,
-    promptDomains,
-  }
-}
+export default namedConnect(
+  (state, {conversationIDKey, ordinal}: OwnProps) => {
+    const message = Constants.getMessage(state, conversationIDKey, ordinal)
+    const messageID = message && message.type === 'text' ? message.id : noMessageID
+    let promptDomains: Set<string> | undefined
 
-const mapDispatchToProps = (dispatch, {conversationIDKey}: OwnProps) => ({
-  _setPolicy: (messageID: Types.MessageID, domain: string, result: RPCChatTypes.UnfurlPromptResult) => {
-    dispatch(
-      Chat2Gen.createUnfurlResolvePrompt({
-        conversationIDKey,
-        domain,
-        messageID,
-        result,
-      })
-    )
+    const pm = state.chat2.unfurlPromptMap.get(conversationIDKey)
+    if (pm) {
+      promptDomains = pm.get(messageID)
+    }
+    return {
+      messageID,
+      promptDomains: promptDomains || noPrompts,
+    }
   },
-})
-
-const makeRes = (actionType: RPCChatTypes.UnfurlPromptAction, domain?: string) => {
```

Comment:
this wasn't typed correctly. the `onetime`/`accept` fields are only acceptable on those `actionType`s